tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "jsx": "preserve"
   },
   "include": ["src"],
```

Comment:
The next PR @David-Cohen35 we'll want to think about how we want to change this `includes` key. We can either remove it, specify a dot maybe, or add additional folders on top of src to track.